.github/workflows/release.yml

```diff
@@ -74,11 +74,6 @@ jobs:
           poetry run pyinstaller operate/services/utils/tendermint.py --onefile
           cp dist/tendermint dist/tendermint_bin
 
-          # patch open aea in place
-          rm -fr ./open-aea
-          git clone https://github.com/valory-xyz/open-aea.git -b fix/1.5.2_encoding
-          poetry run pip install ./open-aea/
-
```

Comment:
@solarw please confirm if it is fine to remove this (i.e., the encoding is already fixed in recent releases of the repository).

Reply:
@jmoreira-valory yes, it's good to remove